README.md

```diff
@@ -65,9 +61,28 @@ See documentation in the test suite
 
 The rule definition index is [here](docs/README.md).
 
-## Change Management
+## Proposed Changes
+
+We actively accept, and encourage, pull requests for code changes. A list of the requirements for a pull request follows, and the request will be reviewed by the technical leads of the project. If the request is accepted it will be merged into the appropriate branch. Some requests may require Committee approval which may take longer to implement. If the request is found to be missing parts or is otherwise incomplete, comments will be noted regarding the missing or incomplete parts.
+
+### Requirements for a Pull Request(PR):
+
+  - Branch off master, develop on your independent fork, PR back to master or other appropriate branch on the root fork.
+  - Your code should pass [flake8](https://github.com/bmcustodio/flake8).
+  - Unit test coverage is required or an explanation for why the change is already covered or not coverable.
+  - Good [Docstrings](https://github.com/Workiva/styleguide/blob/master/PYTHON.rst#docstrings) are required.
+  - Good [commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) are required.
+  - The pull request must have at least one `+1` from another community member.
+  - There must be no unaddressed comments.
+  - PR text must be in the specified format:
+    ```
```

Comment:
Yeah we can look into it for a follow up. I wanted to get the template out there ASAP, but I will experiment with the auto-PR formatting as well and see if I can cut a followup PR.